docs/current/sailbot_workspace/launch_files.md

```diff
@@ -0,0 +1,175 @@
+
+    - To switch between the global process and package launch process diagrams, hover over the image to make
+      the toolbar visible and navigate with the arrows on the left side
+
+<!-- This was generated by going to the source, selecting File > Embed > iFrame, and deselecting Layers, Tags, and Edit -->
+<!-- Source: https://drive.google.com/file/d/1B5ayOHnFcVvFmLeAGr6nOryFvVqk5abI/view?usp=sharing -->
+<iframe frameborder="0" style="width:100%;height:600px;" src="https://viewer.diagrams.net/?highlight=0000ff&nav=1&title=Launch_File_Architecture.drawio#Uhttps%3A%2F%2Fdrive.google.com%2Fuc%3Fid%3D1B5ayOHnFcVvFmLeAGr6nOryFvVqk5abI%26export%3Ddownload"></iframe>
+
+### The Global Launch Process
+
+The global launch process is intended to start up the entire system (both the development and production environments).
+This process invokes the package launch files for each ROS package used in the system through a global launch file.
+The process is as follows:
+
+1. The global launch file is invoked with the user passing arguments via the CLI and specifying a configuration file.
+2. Environment variables common to all ROS packages are declared. In addition, the global arguments common across
+all ROS packages are declared.
+3. For each package launch file:
+    - The CLI arguments, global argument declarations, and environment variables are passed into the package launch
+      file.
+    - Local arguments, specific to the package, are declared.
+    - Both the global and local arguments are parsed based on the argument declarations and are set for use upon
+      start up.
+    - The environment variables are set for use upon start up.
+    - The ROS nodes belonging to the package begin execution, utilizing the ROS parameters from the configuration file.
+
+### The Package Launch Process
+
+The package launch process is intended to start up a specific ROS package by directly using the package launch file.
+The process is as follows:
+
+1. The package launch file is invoked with the user passing arguments via the CLI and specifying a configuration file.
+2. Global argument declarations and environment variables are loaded into the launch process.
+3. Local arguments, specific to the package, are declared.
+4. Both global and local arguments are parsed based on the argument declarations and are set for use upon start up.
+5. The environment variables are set for use upon start up.
+6. The ROS nodes belonging to the package begin execution, utilizing the ROS parameters from the configuration file.
+
+??? warning "When launching individual packages, be aware of dependencies between ROS packages"
+    Some packages rely on the data produced by other packages in the system. This may cause only
+    partial functionality of the ROS node(s) that are running inside the launched package. Therefore,
+    it may be necessary to launch multiple packages manually to get the desired functionality.
+
+## Invoking Launch Files
```

Comment:
Maybe we can refer to this section directly in the workflow page to avoid duplicate information